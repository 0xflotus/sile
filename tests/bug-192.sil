\begin[papersize=a5]{document}\nofolios
\font[family=Noto Naskh Arabic,language=urd,size=18pt]
 {}\break
\skip[height=17cm]
کب سے ہیں اِسی کوشش میں اب\break\skip[height=2ex]
کب سے ہیں اِسی کوشش میں اب\break
\end{document}
