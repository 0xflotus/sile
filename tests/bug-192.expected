<tests/bug-192.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.97638
My 	528.45355
Set font 	Noto Naskh Arabic;14;400;Regular;normal;;RTL
T	40 39	(اب)
Mx 	58.85065
T	365 408 495	(میں)
Mx 	82.20733
T	461 462 503 354	(کوشش)
Mx 	121.74469
T	406 459 80 39	(اِسی)
Mx 	147.63751
T	365 408 369	(ہیں)
Mx 	168.47174
T	374 459	(سے)
Mx 	188.83429
T	435 354	(کب)
[1] New page
Mx 	40.97638
My 	39.83995
T	40 39	(اب)
Mx 	58.85065
T	365 408 495	(میں)
Mx 	82.20733
T	461 462 503 354	(کوشش)
Mx 	121.74469
T	406 459 80 39	(اِسی)
Mx 	147.63751
T	365 408 369	(ہیں)
Mx 	168.47174
T	374 459	(سے)
Mx 	188.83429
T	435 354	(کب)
[2] End page
Finish

